target-platforms/packages/dxp-7.4/package.json

```diff
@@ -147,16 +162,23 @@
 		"metal-useragent": "3.0.1",
 		"metal-web-component": "2.16.8",
 		"prop-types": "15.7.2",
+		"qrcode": "1.4.4",
+		"qs": "6.7.0",
 		"querystring": "0.2.0",
 		"react": "16.12.0",
 		"react-dnd": "11.1.1",
 		"react-dnd-html5-backend": "11.1.1",
 		"react-dom": "16.12.0",
+		"react-text-mask": "5.4.3",
+		"react-transition-group": "4.4.1",
 		"recharts": "1.8.5",
 		"senna": "2.7.9",
+		"text-mask-addons": "3.8.0",
+		"text-mask-core": "5.1.2",
+		"uuid": "9.0.0",
 		"xss-filters": "1.2.7"
 	},
-	"description": "Target platform for Liferay Portal 7.4.13-u4",
+	"description": "Target platform for Liferay Portal 7.4.13-u66",
```

Comment:
whoa, quite the diff here

Reply:
Yeah, that's because we don't release automatically yet and because, for now (without formal APIs) any version is valid for any customer.